python/guide/module/gluon/normalization.md

```diff
@@ -0,0 +1,412 @@
+# Normalization Blocks
+
+When training deep neural networks there are a number of techniques that are thought to be essential for model convergence. One important area is deciding how to initialize the parameters of the network. Using techniques such as Xavier initialization, we can can improve the gradient flow through the network at the start of training. Another important technique is normalization: i.e. scaling and shifting certain values towards a distribution with a mean of 0 (i.e. zero centered) and a standard distribution of 1 (i.e. unit variance). Which values you normalize depends on the exact method used as we'll see later on.
```

Comment:
Suggestion: When training deep neural networks .... Model convergence significantly depends on how you initialize the parameters of the network. Using techniques such as ..... We can also use a technique called normalization: i.e.